docs/Android/android-install.md

```diff
@@ -39,7 +39,7 @@ def ws1IntelSdkVersion = "24.3.1"
 
```

Comment:
Can we hold this PR until we have a release out with a different package name please?